docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,777 @@
+
+A community contribution from [@mangeg](https://github.com/mangeg) improved support for ANSI escape codes and removed this limitation. Behold:
+
+![Colorful console logs](images/console-logs-ansi-text-format.png)
+
+Another improvement to console logs is hiding unsupported escape codes. Codes that aren't related to displaying text, such as positioning the cursor or communicating with the operating system don't make sense in this UI, and are hidden.
+
+## Telemetry
+
+### Improve telemetry filtering
+
+Traces can now be filtered with attribute values. For example, if you only want to view traces for one endpoint in your app, the `http.route` attribute on HTTP requests can be filtered to a specified value.
+
+Telemetry filtering also supports auto-complete of existing values. The **Add filter** dialog provides a combo box for selecting from values that dashboard has received. This feature makes it much easier to filter to real data and helps avoid typos by entered a value yourself.
+
+<video controls src="videos/trace-filter.mp4" title="Video of filtering traces"></video>
+
+### Combine telemetry from multiple resources
+
+It's now possible to filter telmetry in the dashboard to all instances of a resource. This is useful if you have multiple replicas for an app and want to view their telemetry together.
+
+To view telemetry for all instances of a resource, select the top-level option in the resource selector that's marked with `(application)`:
+
+![Filter by all instances of a resource](images/telemetry-resource-filter.png)
+
+### Browser telemetry support
+
+The dashboard now supports OpenTelemetry Protocol (OTLP) over HTTP and cross-origin resource sharing (CORS). These features unlock the ability to send OpenTelemetry from browser apps to the dashbaord.
+
+For example, a browser-based single page app (SPA) can configure the [JavaScript OTEL SDK](https://opentelemetry.io/docs/languages/js/getting-started/browser/) to send structured logs, traces and metrics created in the browser to the dashboard. Browser telemetry is displayed alongside server telemetry.
+
+![Trace detail page with browser telemetry](images/dashboard-browser-telemetry.png)
+
+For more information on configuring browser telemetry, see [Enable browser telemetry](https://learn.microsoft.com/dotnet/aspire/fundamentals/dashboard/enable-browser-telemetry) documentation.
+
+## App Host (Orchestration)
+
+### Waiting for dependencies
+
+The app host now supports waiting for dependencies to be ready before starting another resource. This is useful when a resource is taking a long time to start up. The app host will wait for the resource to be ready before starting any dependent resources.
```

Comment:
While this is an app host feature, I think we can talk about it more in terms of what the user can now do. Something like:

```suggestion
You can now specify that a resource should wait for another resource before starting. This can help avoid connection errors during startup by only starting resources when their dependencies are ready.
```

I removed the mention of long startup time as I figure a race condition is a race condition, regardless of the duration.